docs/running-on-kubernetes.md

```diff
@@ -1662,6 +1662,28 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>3.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.ui.custom.driver.log.url</code></td>
```

Comment:
Please remove this from this table. You can add a paragraph at `Accessing Logs` section like `spark.ui.custom.executor.log.url`

Reply:
done